packages/dds/tree/src/feature-libraries/chunked-forest/codec/codecs.ts

```diff
@@ -38,6 +39,13 @@ export type FieldBatchCodec = IJsonCodec<
 	FieldBatchEncodingContext
 >;
 
+/**
+ * TODO: makeFieldBatchCodec (and makeVersionedValidatedCodec transitively) should bake in this versionToFormat logic and the resulting codec can then support use with FluidClientVersion directly.
```

Comment:
Can we document what this is/does as well? Presumably the TODO should go in a `@privateRemarks`

Reply:
Renamed and documented.